pkg/http/rest/api.go

```diff
@@ -7,16 +7,21 @@ import (
 	"errors"
 	"fmt"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/configuration"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/helpers"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/log"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/status"
 	"net/http"
-	"strings"
 )
 
 type CreateBodyFN func() []byte
 type GetEndpointFN func(string) string
 type JsonObjectFN func() interface{}
 
+type JiraAPIInterface interface {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type JiraAPIInterface should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4777494)